### ISSUE TITLE
ERROR: 'content_html' when downloading a playlist

### ISSUE THREAD

**Issue by Alxiice** [michaelb/auto-ytdl#23]
URL of the playlist : https://www.youtube.com/playlist?list=PLB5ialYt3fSRgMsQhKsy634InY1PKeX5I
Command : python3 aytdl.py update https://www.youtube.com/playlist\?list\=PLB5ialYt3fSRgMsQhKsy634InY1PKeX5I -i -f

**Comment by michaelb** [comment]
i can't replicate, do you use the latest youtube-dl version? Mine is 2020.09.14

(what is the output of `youtube-dl  --version`?)

**Comment by Alxiice** [comment]
Ok I fixed this but now I can only download 36 videos (out of more than a hundred) before having an error :
```
Error code: 256
youtube-dl encountered at least one error.
That can happen when a video you are trying to download is not-standart (geo-restricted, not released yet, 'première').
```
I also have a traceback :
```
Traceback (most recent call last):
  File "aytdl.py", line 18, in <module>
    main()
  File "/home/selxis/Bureau/auto-ytdl/autoytdl/AYTDL.py", line 222, in main
    a.move_to_library()
  File "/home/selxis/Bureau/auto-ytdl/autoytdl/AYTDL.py", line 121, in move_to_library
    if should_add(temp_dir_path+"/"+filename, self.config):
  File "/home/selxis/Bureau/auto-ytdl/autoytdl/metadata_manager.py", line 108, in should_add
    if inlist(tuple_audio, list_tags):
  File "/home/selxis/Bureau/auto-ytdl/autoytdl/metadata_manager.py", line 84, in inlist
    if equal(eval(x), audio_tuple):
  File "/home/selxis/Bureau/auto-ytdl/autoytdl/metadata_manager.py", line 45, in equal
    if abs(float(m2[2]) - float(m1[2])) < 15:
ValueError: could not convert string to float: ''
```

**Comment by michaelb** [comment]
That's another problem that should have warranted another issue, but it was simple to fix (fixed via #25 now in master v1.2.2)

The root of the problem is that this particular video has no length. It's a bit weird to say, but this means either the otherwise perfectly working metadata python module broke over this particular file, or the original file hosted on youtube is somehow corrupted. 
Given there is a handful of videos in that playlist that looks weird and fail to be downloaded via youtube-dl, i'd bet on the later.

Either way it's probably not something I'd want in my music library, but i get that it may only be a youtube-dl/python module bug, so i fixed this particular issue. I was able to download this whole playlist without any further issues.


Mind you, aytdl limits the size of individual playlist/channels downloads in order to not mistakenly download 10k videos, which could be disastrous. You should run `aytdl edit` (or `python aytdl.py edit` if you're running from source) to edit the config file and raise that limit
